configure/stack_example.json

```diff
@@ -3,48 +3,36 @@
         {
             "name": "vRack1",
             "vPDU": [
-                {
-                    "name": "vpdu_1",
-                    "community": "foo",
-                    "ip": "172.31.128.1",
-                    "outlet": {
-                        "1.1": "bar",
-                        "1.2": "bar",
-                        "1.3": "bar"
-                    }
-                }
```

Comment:
@Summer-Qing simon, this stack example is a demonstration on how user compose a logical stack.
Though we've not introduce vPDU application yet, this interface is still needed for others to refer to.
Please keep this part of codes.

Other code change, i.e. rename node, in this json example is good.